packages/types/src/transport.ts

```diff
@@ -14,7 +16,9 @@ export type TransportMakeRequestResponse = {
 
 export interface InternalBaseTransportOptions {
   bufferSize?: number;
+  recordDroppedEvent?: (reason: EventDropReason, dataCategory: DataCategory) => void;
```

Comment:
Should we just make this required?

Reply:
As discussed: yeah we should as clients should always pass this in